raincloudy/faucet.py

```diff
@@ -219,12 +219,24 @@ def _set_watering_time(self, zoneid, value):
         ddata[attr] = value
         self.submit_action(ddata)
 
+    # TODO in a future release we should break this out. artifact of old API
     @property
     def watering_time(self):
```

Comment:
This property really should be 2 properties now. but I'm trying to minimize Home Assistant changes at this point yet. The change in this PR should make the property behave as it did pre-API change